product_barcode_generator/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Product barcode generator
+=========================
+
+This module will add a function which leads to an automatic generation of EAN13 for products
+
+You will have to define the company default value (6 firsts number of EAN13) then the 6 next number the sequence.
```

Comment:
s/6 firsts number/first 6 numbers/
s/6 next number/next 6 numbers/

**of** the sequence?